content/topics/java-specific/concurrency/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+content_type: topic
+ready: true
+tags:
+- threads
+- concurrency
+title: Java Multithread, Concurrency and Parallelism
+---
+
+
+I am sure by now you have heard about java being multi-threaded, what does that mean exactly
+
+Unlike for example JavaScript which is a single threaded language Java on the other has multiple threads to help you run your program faster. 
```

Comment:
Hmm, this one also needs revision...

There's a trick I use when writing that might help you.  When I write, I imagine what the words will sound like if I read them aloud.  The sentence above has no punctuation, it would be spoken in a single rushed breath.